packages/wonder-blocks-data/components/data.md

```diff
@@ -17,29 +17,11 @@ data or an error, we re-render.
 ```jsx
 import {Body, BodyMonospace} from "@khanacademy/wonder-blocks-typography";
 import {View} from "@khanacademy/wonder-blocks-core";
-import {initializeCache, Data, RequestHandler} from "@khanacademy/wonder-blocks-data";
```

Comment:
If it's not in the docs, can we remove it from the public API?

Reply:
What exactly?

Reply:
Sorry... I was so focused on `initializeCache` I didn't realize there were other imports on this line that also got removed. 🙃 

I was wondering if `initializeCache` still needs to be part of the public API, but now that I ask this question I realize it does otherwise how will we do the 2nd pass of SSR-ing. 😅

Reply:
Yeah, we need the `initializeCache` call so we can initialize the cache before client-side rendering.